CDAlertView/Classes/CDAlertView.swift

```diff
@@ -532,7 +555,9 @@ open class CDAlertView: UIView {
                                             cornerRadii: CGSize(width: 8.0, height: 8.0))
         let roundLayer = CAShapeLayer()
         roundLayer.path = roundCornersPath.cgPath
-        buttonView.layer.mask = roundLayer
+        if hasRoundCorners == true {
```

Comment:
I think it's better to include `roundCornersPath`, `roundLayer` inside the `if` statement. There is no need for them outside of the `if` statement.

Reply:
Sounds good.